lib/const.rb

```diff
@@ -35,11 +35,13 @@
   HOME = CREW_PREFIX + ENV['HOME']
 end
 
+# File.join ensures a trailing slash if one does not exist.
 if ENV['CREW_CACHE_DIR'].to_s == ''
```

Comment:
`if ENV['CREW_CACHE_DIR'].to_s.empty`?

Reply:
Sure, but that was the existing code...

Reply:
I changed all  `.to_s == ''` to `.to_s.empty` .